lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -1063,6 +1109,26 @@ estimateTxSize skeleton =
         sizeOf_ChainCode  = 34
         sizeOf_Attributes = 45 -- NOTE: could be smaller by ~34 for Icarus
 
+    -- native_script =
+    --   [ script_pubkey      = (0, addr_keyhash)
+    --   // script_all        = (1, [ * native_script ])
+    --   // script_any        = (2, [ * native_script ])
+    --   // script_n_of_k     = (3, n: uint, [ * native_script ])
+    --   // invalid_before    = (4, uint)
+    --      ; Timelock validity intervals are half-open intervals [a, b).
+    --      ; This field specifies the left (included) endpoint a.
+    --   // invalid_hereafter = (5, uint)
+    --      ; Timelock validity intervals are half-open intervals [a, b).
+    --      ; This field specifies the right (excluded) endpoint b.
+    --   ]
+    sizeOf_NativeScript = \case
```

Comment:
Good to add this into the size estimation, because I think it would also be needed for spending from a multi-sig wallet.